lib/internal/crypto/webcrypto.js

```diff
@@ -923,7 +924,153 @@ class SubtleCrypto {
   constructor() {
     throw new ERR_ILLEGAL_CONSTRUCTOR();
   }
+
+  static supports(operation, algorithm, lengthOrAdditionalAlgorithm = null) {
```

Comment:
A comment here that explains the heuristics or points to the spec would be good as a reminder that this is implementing to a spec.